lib/selector.py

```diff
@@ -20,7 +20,14 @@ def getprop(obj,path,keyErrorAsNone=False):
         else:
             return None
 
-    return getprop(obj[pp],pn,keyErrorAsNone)
+    if exists(obj, pp):
```

Comment:
I'm confused, isn't this an `else` condition on the `if` statement above?

Reply:
Yes, that should be an else on `if exists`